mk_verificator/tests/networking/mtu/gauges.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
I need help on that and I will do that in next pull request as it is right now already very big. Can we merge that without those changes?